aea/contracts/__init__.py

```diff
@@ -18,3 +18,7 @@
 # ------------------------------------------------------------------------------
 
 """This module contains the contract modules."""
+from aea.contracts.base import Contract
+from aea.crypto.registries import Registry
+
+contract_registry: Registry[Contract] = Registry[Contract]()
```

Comment:
using a global registry. A potential issue when having many agents with the same contract but different configurations.